lib/aggregation/accumulator/src/index.js

```diff
@@ -351,9 +351,13 @@ const runCLI = () => {
 
   // Create the app, replay any failed events, and listen
   const app = accumulator();
-  dataflow.replay(app.reducer, 0, () => {
-    app.listen();
-  });
+  app.listen();
+  if (cluster.isWorker()) {
+    debug('Replaying for accumulator');
+    dataflow.replay(app.reducer, 0, () => {
+      debug('Replay finished successfully for accumulator');
```

Comment:
We need to check for error. Maybe something like:
```
dataflow.replay(app.reducer, 0, (err) => {
  debug('Replay completed %s', err ? 'with error' : 'successfully');
});
```	

The error and the docs are already logged by dataflow.